Content.Server/Bible/Components/SummonableComponent.cs

```diff
@@ -31,6 +31,14 @@ public sealed partial class SummonableComponent : Component
         [DataField("requiresBibleUser")]
         public bool RequiresBibleUser = true;
 
+        // DeltaV - Begin Anomalite Limited Respawns
+        [DataField("lives")]
+        public int? Lives = null;
+        public int Deaths = 0;
```

Comment:
Blank line
Also Deaths should probably also be a DataField
```suggestion
        public int? Lives = null;

        [DataField]
        public int Deaths = 0;
```

Reply:
The blank space is to separate the fields from the calculated value, ill add that deaths field though, i dont expect it to be used as a yml field though.

Reply:
> [...] ill add that deaths field though, i dont expect it to be used as a yml field though.

Right, instead of DataField could do ViewVariable ReadWrite, which is probably more in line with what I wanted anyways

Reply:
Implemented as suggested.